src/devices/machine/k056230.cpp

```diff
@@ -2,7 +2,24 @@
 // copyright-holders:Fabio Priuli
 /***************************************************************************
 
-    Konami IC 056230 (LANC)
+Konami IC 056230 (LANC)
+
+Schematic (located on page 28): http://arcarc.xmission.com/PDF_Arcade_Manuals_and_Schematics/Racing_Force_Schematics.pdf
+
```

Comment:
(C) Konami, probably not a good idea to link in source.
In fact it says "PROPRIETARY AND CONFIDENTIAL"